source/partials/_head.html.erb

```diff
@@ -5,3 +5,16 @@
 <title><%= current_page.data.title %></title>
 <%= stylesheet_link_tag "main", {media: false} %>
 <meta name="description" content="<%= current_page.data.description %>" />
+<meta property="og:locale" content="en_US">
+<meta property="og:title" content="#ClimateStrike Poster & Flyer Maker">
+<meta property="og:description" content="Make your own customizable, downloadable poster or flyer for your #climatestrike event.">
```

Comment:
Could this pull from `current_page.data.description` like the `description` tag above?

Reply:
definitely! I did this quick and dirty since folks were already starting to share it, but for the future it would be great to separate this content from the template.

Reply:
Would that be a helpful pull request? Happy to put something up if you like.